src/charm.py

```diff
@@ -1125,8 +1138,12 @@ def _apache_disable_config(self, conf_name):
         self._run_cli(["a2disconf", conf_name])
         self._start_server()
 
-    def _plugin_swift_reconciliation(self):
-        """Reconciliation process for swift object storage (openstack-objectstorage-k8s) plugin."""
+    def _swift_config(self):
+        """Load swift configuration from charm config.
+
+        Returns:
+            Swift configuration in dict.
+        """
         swift_config_str = self.model.config["wp_plugin_openstack-objectstorage_config"]
         swift_config_key = [
```

Comment:
If I understand well, there are the mandatory keys in swift_config ?
If so, I'd rename the variable to something more explicit like `mandatory_swift_config_key`

Reply:
The keys listed here is required for the WordPress OpenStack swift plugin, they all have to exist in the config for now, but I am planning to add some default values in the future.

Reply:
What about renaming the variable then ?

Reply:
Renamed to `required_swift_config_key`, thanks!